API_Kraken/get_bitcon_price.py

```diff
@@ -0,0 +1,19 @@
+import requests
+import datetime
+
+get_request = requests.get('https://api.kraken.com/0/public/Ticker?pair=XBTUSD')
+price = get_request.json()['result']['XXBTZUSD']['a'][0]
+date = datetime.datetime.now().strftime("%c")
+value = ('Price of bitcon for: ' + date + ' is: ' + price + ' USD')
+try:
+    print(value)
+except:
+    print("An error has occured")
+
+value_txt = {"Price of bitcon for:" : date, "is" : price}
+
+file = open("bitcoin_price.txt", "w")
```

Comment:
This is risky as if there will be exception raised before  we close file then it will remain locked and ext execution of script will fail. 

therefore it is safer to use autoclosable faeture:

https://stackoverflow.com/questions/45954214/python-automatically-closing
```
 with open("bitcoin_price.txt", "a") as myfile:
     
     myfile.write(...)
```
Above file will be automatically closed when we finish script or if there is any exception